suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -92,12 +87,17 @@ export const XpubScanScreen = ({
     };
 
     return (
-        <Screen header={<AccountImportHeader activeStep={1} />}>
+        <Screen header={<AccountImportHeader activeStep={2} />}>
             <Box>
```

Comment:
maybe this one as well - that was probably me 🥶

Reply:
removed e21e5713ac754d22544aedeb5ae901093cf708a7